modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryListeningTimeView.kt

```diff
@@ -84,42 +106,66 @@ private fun SecondaryText(
     story: StoryListeningTime,
     modifier: Modifier,
 ) {
-    val context = LocalContext.current
-    val funnyText = FunnyTimeConverter().timeSecsToFunnyText(
-        story.listeningTimeInSecs,
-        context.resources
-    )
-    StorySecondaryText(text = funnyText, color = story.subtitleColor, modifier = modifier)
+    val text = stringResource(LR.string.end_of_year_listening_time_subtitle)
+    StorySecondaryText(text = text, color = story.subtitleColor, modifier = modifier)
 }
 
 @Composable
-private fun PodcastCoverRow(
-    story: StoryListeningTime,
-    modifier: Modifier = Modifier,
-) {
+private fun ListenedTimeTexts(story: StoryListeningTime) {
     val context = LocalContext.current
-    val currentLocalView = LocalView.current
-    val coverWidth = (currentLocalView.width.pxToDp(context).dp - 15.dp) / 3
-    Box(
-        modifier = modifier
-            .graphicsLayer { translationY = coverWidth.toPx() / 1.75f }
-            .height(coverWidth * 2.2f)
+    val listeningTimeDisplayStrings = getListeningTimeDisplayStrings(context, story.listeningTimeInSecs)
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.fillMaxWidth(),
     ) {
-        Row(
-            modifier
-                .transformPodcastCover()
-        ) {
-            listOf(1, 0, 2).forEach { index ->
-                val podcastIndex = index.coerceAtMost(story.podcasts.size - 1)
-                Row {
-                    PodcastCover(
-                        uuid = story.podcasts[podcastIndex].uuid,
-                        coverWidth = coverWidth,
-                        coverSize = CoverSize.SMALL
+        Text(
+            text = listeningTimeDisplayStrings.firstNumber,
+            color = story.tintColor,
+            fontSize = ListenedTimeFontSize.nonScaledSp,
+            lineHeight = ListenedTimeFontSize.nonScaledSp,
+            fontWeight = FontWeight.W300,
+            fontFamily = StoryFontFamily,
+            style = LocalTextStyle.current.merge(
+                @Suppress("DEPRECATION")
+                TextStyle(
+                    platformStyle = PlatformTextStyle(
+                        includeFontPadding = false,
+                    ),
+                    lineHeightStyle = LineHeightStyle(
+                        alignment = LineHeightStyle.Alignment.Top,
+                        trim = LineHeightStyle.Trim.Both
```

Comment:
I tried reducing font padding between listening time and its subtitle but could not reproduce the gap between them for large font size. I even tried to limit line height but it didn't seem to make any difference.

I'm using `100` font size as compared to the font size of `278` in Figma, but feel free to suggest any improvement.

Reply:
I added it as a sub-task in the main project thread under refinements. I'll revisit it later.